CMakeLists.txt

```diff
@@ -34,6 +35,16 @@ if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
         "Install systemd service file to a directory manually overridable with Systemd_SERVICES_INSTALL_DIR"
         OFF "NOT GUI" OFF
     )
+    check_cxx_source_compiles(
```

Comment:
Why not place it inside `if (STACKTRACE)` to avoid unneeded checking if stacktrace is disabled?

Reply:
@glassez Good point, fixed.